test/e2e/persistent_volume/persistent_volume_test.go

```diff
@@ -133,15 +134,16 @@ var _ = ginkgo.Describe("PersistentVolume", func() {
 		Ω(err).NotTo(HaveOccurred())
 	})
 
+	saName := "nfs-service-account"
+	crName := "nfs-cluster-role"
+	crbName := "nfs-cluster-role-binding" //nolint:gosec
+	serverName := "nfs-provisioner"
+	scName := "nfs-sc"
```

Comment:
nit:  Could we use const to encapsulation those variables and move to the beginning of the test file to align with this test's coding style. ([e2e/recovery_and_restart/recovery_and_restart_test.go#L39-L48](https://github.com/apache/yunikorn-k8shim/blob/a53123f2d0bb3fd325dd7f8c1f2cc4640ff448d7/test/e2e/recovery_and_restart/recovery_and_restart_test.go#L39-L48))

```suggestion
const (
	saName = "nfs-service-account"
	crName = "nfs-cluster-role"
	crbName = "nfs-cluster-role-binding" //nolint:gosec
	serverName = "nfs-provisioner"
	scName = "nfs-sc"
)
```

Reply:
Thanks @chenyulin0719 for helping me review! I put them to the begin of the test file and keep align the naming style with other const